examples/slash_commands/main.go

```diff
@@ -266,13 +264,12 @@ var (
 				},
 			})
 		},
-		"responses": func(s *discordgo.Session, i *discordgo.InteractionCreate) {
-			// Responses to a command are very important.
-			// First of all, because you need to react to the interaction
-			// by sending the response in 3 seconds after receiving, otherwise
-			// interaction will be considered invalid and you can no longer
-			// use the interaction token and ID for responding to the user's request
 
+		// The Initial Response to an Interaction must occur within 3 seconds of recieving it.
+		// Otherwise the interaction token will be invalidated (disables the interaction token and ID).
+		// Valid interaction tokens last for 15 minutes.
```

Comment:
We must clarify that they are valid because invalid interaction tokens (that result from a lack of response within 3 seconsd) do not last for 15 minutes.